src/Raven.Server/Web/Authentication/OAuthApiKeyHandler.cs

```diff
@@ -68,15 +69,16 @@ public class OAuthApiKeyHandler : RequestHandler
                     }
                     catch (Exception e)
                     {
-                        if (Logger.IsWarnEnabled)
-                            Logger.WarnException("Failed to authenticate api key", e);
+                        if (_logger.IsOperationsEnabled)
+                            _logger.Operations("Failed to authenticate api key", e);
```

Comment:
This isn't an operations level issue.
Old API key / revoke/etc can happen all the time, and they would spam the operations logs